RELEASE_NOTES.md

```diff
@@ -1,5 +1,130 @@
+- A new command, `podmansh`, has been added, which executes a user shell within a container when the user logs into the system. The container that the users get added to can be defined via a Podman Quadlet file.
+- The `podman network create` command supports a new `--option`, `bclim`, for the `macvlan` driver.
+- The `podman info` command now prints network information about the binary path, package version, program version and DNS information ([#18443](https://github.com/containers/podman/issues/18443)).
+- The `podman info` command now displays the number of free locks available, helping to debug lock exhaustion scenarios.
+- The `podman info` command now outputs information about pasta, if it exists in helper_binaries_dir or $PATH.
+- The remote Podman client’s `podman build` command now accepts Containerfiles that are not in the context directory ([#18239](https://github.com/containers/podman/issues/18239)).
+- The remote Podman client’s `podman play kube` command now supports the `--configmap` option ([#17513](https://github.com/containers/podman/issues/17513)).
+- The `podman kube play` command now supports multi-doc YAML files for configmap arguments. ([#18537](https://github.com/containers/podman/issues/18537)).
+- The `podman pod create` command now supports a new flag, `--restart`, which sets the restart policy for all the containers in a pod.
+- The `--format={{.Restarts}}` option to the `podman ps` command now shows the number of times a container has been restarted based on its restart policy.
+- The `--format={{.Restarts}}` option to the `podman pod ps` command now shows the total number of container restarts in a pod.
+- The podman machine provider can now be specified via the `CONTAINERS_MACHINE_PROVIDER` environment variable, as well as via the `provider` field in `containers.conf` ([#17116](https://github.com/containers/podman/issues/17116)).
+- A default list of pasta arguments can now be set in `containers.conf` via `pasta_options`.
+- The `podman machine init` and `podman machine set` commands now support a new option, `--user-mode-networking`, which improves interops with VPN configs that drop traffic from WSL networking, on Windows.
+- The remote Podman client’s `podman push` command now supports the `--digestfile` option ([#18216](https://github.com/containers/podman/issues/18216)).
+- Podman now supports a new option, `--out`, that allows redirection or suppression of STDOUT ([#18120](https://github.com/containers/podman/issues/18120)).
+
+### Changes
+- The `--filter id=xxx` option will now treat `xxx` as a CID prefix, and not as a regular expression ([#18471](https://github.com/containers/podman/issues/18471)).
+- The `--filter` option now requires multiple `--filter` flags to specify multiple filters. It will not longer support the comma syntax (`--filter label=a,label=b`).
```

Comment:
```suggestion
- The `--filter` option now requires multiple `--filter` flags to specify multiple filters. It will no longer support the comma syntax (`--filter label=a,label=b`).
```